GX-AnalyticContinuation/pade_approximant.f90

```diff
@@ -0,0 +1,169 @@
+! COPYRIGHT
```

Comment:
Are we including the Pade interpolation on this PR?

Reply:
No, I've scrapped it from the branch